docs/power-payment-app.md

```diff
@@ -0,0 +1,127 @@
+# Power Payment App
+
+## Overview
+
+**What does it do?**
+
+PowerPay is a mobile payment application that allows users to send and receive money, top-up their accounts, and withdraw cash. It offers a flexible payment experience by utilizing both smartphones and physical devices called Choronko.
+
+**Who is using it?**
+
+- PowerPay Users: Individuals with smartphones who can initiate and manage transactions through the app.
+- PowerPay Kiosks: Physical kiosks that facilitate top-ups and withdrawals for users with smartphones and Choronko.
+- Choronko Users: Individuals who can send and receive payments using the Choronko device without needing a smartphone.
+
+## Technologies used
+
+- ReactJS for building the user interface and implementing a Progressive Web App (PWA) approach.
+- HTML and CSS for the structure and styling of the user interface.
+- RESTful APIs for communication with the backend server.
+
```

Comment:
Apart from these, we'll also have other technologies like react libraries (like navigation, state management,...), PWA and HTTP communications, under which RESTFul APIs are done